sunpy/map/sources/tests/test_eit_source.py

```diff
@@ -5,25 +5,37 @@
 
 import astropy.units as u
 
-from sunpy.data.test import get_dummy_map_from_header, get_test_data_filenames, get_test_filepath
+import sunpy.map
+from sunpy.data.test import get_dummy_map_from_header, get_test_filepath
 from sunpy.map.sources.soho import EITMap
-
-header_list = [f for f in get_test_data_filenames() if 'efz' in f.name and '.header' in f.name]
+from . import _test_private_date_setters
 
 __author__ = "Pritish C. (VaticanCameos)"
 
 
-@pytest.fixture(scope="module", params=header_list)
-def eit_map(request):
+@pytest.fixture()
+def eit_map():
     """Creates an EITMap from a FITS file."""
-    return get_dummy_map_from_header(request.param)
+    return sunpy.map.Map(get_test_filepath("EIT/efz20040301.000010_s.fits"))
```

Comment:
What's going on here?

Reply:
I need to check that date and reference_date are correct but we have like 6 file headers we run these tests though (for no reason I can see). So I just use the first one.

Reply:
but you have now done this for *all* the tests? This seems like a backwards step? Why not just make the date tests use the single map?

Reply:
Yeah and it doesnt matter. My plan is to remove all of these fits files in a future PR.